src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -686,6 +686,16 @@ private IEnumerable<string> LibrarySearchPaths(string library)
             yield return Path.GetFullPath(library);
         }
 
+        internal static string GetAppDataFolder()
+        {
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+            var dynamoVersion = FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly().Location);
+            var appDataFolder = Path.Combine(Path.Combine(folder, "Dynamo", "Dynamo Core"),
+                $"{dynamoVersion.FileMajorPart}.{dynamoVersion.FileMinorPart}");
```

Comment:
Not sure which path you are trying to build here but I noticed that PathManager.UserDataDirectory has the path to:
C:\Users\roberto.tellez\AppData\Roaming\Dynamo\Dynamo Core\2.17
not sure if is useful.

Reply:
Done.